core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3508,6 +3508,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_REGISTRY_TIMEOUT  =
+      new Builder(Name.WORKER_REGISTRY_TIMEOUT)
+          .setDefaultValue("600000")
```

Comment:
You can change this to `10min` because we will use `ServerConfiguration.getMs(PROPERTY_KEY)` to parse to MS.